charts/README.md

```diff
@@ -70,6 +70,7 @@ Parameter | Description | Default
 `router.svcAddressUpdateTimeout` | The length of update lock expiry time for router to get a service URL returns from executor | `30`
 `router.svcAnnotations` | Annotations for router service | None
 `router.useEncodedPath` | For router to match encoded path. If true, "/foo%2Fbar" will match the path "/{var}"; Otherwise, it will match the path "/foo/bar". | `false`
+`router.traceSamplingRate` | Sample with a rate per time window (traces/second) | `0.5`
```

Comment:
We use `ProbabilitySampler` internally, but the description is about rate-limiting sampler. And looks like the rate-limiting sampler is not implemented yet. In order not to block this PR, how about we modify the description for `ProbabilitySampler` in this PR. We can always switch the sampler once it's implemented.
ref: https://opencensus.io/tracing/sampling/probabilistic/
ref: https://github.com/census-instrumentation/opencensus-go/issues/1178

Reply:
Yes, I see your point. So, what description should we have? Something like that?
> Uniformly sample traces with the given probabilistic sampling rate.

Reply:
yes, i think that would be ok.

Reply:
I've amended the commit. Should be fine now. Thx.